src/routes/main.js

```diff
@@ -10,12 +10,21 @@ async function ctrl(ctx) {
     ctx.body = `Hello World!`
 }
 
+export async function userinfo(ctx) {
+	ctx.body = ctx.state.user
+}
+
 const routes = [
   {
     method: 'get',
     path: '/',
     handler: ctrl
   },
+  {
+    method: 'get',
+    path: '/userinfo',
+    handler: [isAuthenticated, isUnlocked,userinfo]	
```

Comment:
No need for isUnlocked here. isUnlocked is for checking questions. Please test this via postman once

Reply:
@meghprkh  added that by mistake in the second commit. Check now.